test/test_unit.jl

```diff
@@ -15,6 +15,8 @@ using ECOS: Optimizer
 # PERK Single p3 Constructors
 using NLsolve: nlsolve
 
+using SciMLBase: ODEProblem
+
```

Comment:
Why is this added?

Reply:
Ah that is a leftover that I forgot to strip away when removing all relaxation stuff